snomed-ct-integration/pom.xml

```diff
@@ -69,6 +69,12 @@
             <artifactId>data-spinedarray-provider</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
See the comment below -- we don't want to rely on anything from snomedct-transformer since this repo is the replacement. So you can just copy the code you need and remove this dependency.